packages/web-twig/src/Resources/components/FileUploader/FileUploaderInput.twig

```diff
@@ -32,6 +35,9 @@
 {# Attributes #}
 {%- set _nameAttr = _name ? 'name="' ~ _name | escape('html_attr') ~ '"' : null -%}
 {%- set _requiredAttr = _isRequired ? 'required' : null -%}
+{%- set _maxFileSizeAttr = _maxFileSize ? 'data-spirit-max-file-size=' ~ _maxFileSize : null -%}
```

Comment:
I am not sure how we do it in other components, but I have a feeling we prefix these attributes with data if they add data. But please see other components and decide accordingly 
```suggestion
{%- set _dataMaxFileSizeAttr = _maxFileSize ? 'data-spirit-max-file-size=' ~ _maxFileSize : null -%}
```

Reply:
(I am talking only about local variable, the API is ok.)